README.md

```diff
@@ -10,6 +10,7 @@ A friendlier Ruby client for consuming GraphQL-based APIs. Built on top of your
 - [Installation](#installation)
 - [Usage](#usage)
   - [Schema storing and loading on disk](#schema-storing-and-loading-on-disk)
+  - [Preloading schema once](#preloading-schema-once)
```

Comment:
Let's Capitalize Preloading Schema Once, and same for the one above that was inconsistently Capi Talized. Thanks!